src/please.go

```diff
@@ -89,7 +89,7 @@ var opts struct {
 		KeepWorkdirs       bool    `long:"keep_workdirs" description:"Don't clean directories in plz-out/tmp after successfully building targets."`
 		HTTPProxy          cli.URL `long:"http_proxy" env:"HTTP_PROXY" description:"HTTP proxy to use for downloads"`
 		Debug              bool    `long:"debug" description:"When enabled, Please will enter into an interactive debugger when breakpoint() is called during parsing."`
-		KeepGoing          bool    `long:"keep_going" description:"Continue as much as possible after an error. While the target that failed and those that depend on it cannot be build, other prerequisites of these targets can be."`
+		KeepGoing          bool    `long:"keep_going" description:"Continue as much as possible after an error. Builds prerequisites of failed targets and disables safety checks."`
```

Comment:
Im not sure I understand this description. It doesn't only build prerequisites of failed builds. Also not sure what safety checks it disables.